tools/autograd/derivatives.yaml

```
@@ -1297,8 +1297,9 @@
   reserve: not_implemented("batch_norm_backward reserve")
 
 - name: nextafter(Tensor self, Tensor other) -> Tensor
-  self: not_implemented("nextafter")
-  other: not_implemented("nextafter")
+  self: torch.where(self != other, grad, torch.zeros_like(grad))
```

```suggestion
  self: at::where(self != other, grad, 0)
```
Sorry for the confusion here from my other comment. You can't use the python API (torch.*) because this file will eventually be used by cpp codegen. Also, we can avoid materializing the zeros tensor.